src/main/java/qna/domain/BaseTime.java

```diff
@@ -0,0 +1,30 @@
+package qna.domain;
+
+import org.springframework.data.annotation.CreatedDate;
+import org.springframework.data.annotation.LastModifiedDate;
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
+
+import javax.persistence.Column;
+import javax.persistence.EntityListeners;
+import javax.persistence.MappedSuperclass;
+import java.time.LocalDateTime;
+
+@MappedSuperclass
+@EntityListeners(AuditingEntityListener.class)
+public abstract class BaseTime {
```

Comment:
Entity 관련 클래스임을 나타낼수 있도록 다음과 같은 네이밍은 어떨까요? 😄 
```suggestion
public abstract class BaseTimeEntity {
```

Reply:
넵 감사합니다.